components/nexus-repository/src/test/java/org/sonatype/nexus/repository/httpclient/AuthenticationConfigDeserializerTest.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+
+package org.sonatype.nexus.repository.httpclient
+
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import groovy.transform.ToString
+import org.junit.Before
+import org.junit.Test
+import org.sonatype.sisu.litmus.testsupport.TestSupport
+
+/**
+ * Tests for {@link AuthenticationConfigDeserializer}.
+ */
+class AuthenticationConfigDeserializerTest
+    extends TestSupport
+{
+  private ObjectMapper objectMapper
+
+  @Before
```

Comment:
Minor but you should be able to skip the before block and just instantiate as a class member